### PR TITLE
VCR: SearchVCs match exact by default, allow wildcard for notnil/exact match

### PR DESCRIPTION
**Pull request by reinkrul**
Closes https://github.com/nuts-foundation/nuts-node/issues/1619

### CODE REVIEW

File: docs/pages/integrating/vc.rst [hunk 1]
```diff
@@ -118,9 +118,12 @@ The following query does not return all `NutsOrganizationCredential` but **all**
     }
 
 To find certain credentials, you'll need to add fields that are required to exist in the desired credential.
-The following query searches for credentials that have a organization name that starts with an empty string.
-Any credential that does not have an organization name will be ignored.
-By default, field selection is done by matching the given value as prefix.
+By default, matching is exact: it only returns the result when the given value exactly matches.
+There are 2 other matchers for strings:
+- ``"*"`` to match credential fields that contain the field (non-empty)
+- ``*`` as postfix to match credential fields that start with the given string, e.g. ``Hospital Amst*``
```
Comment (woutslakhorst) [inline docs/pages/integrating/vc.rst:124]: Also add that prefix wildcards are not supported, only postfix

Comment (woutslakhorst) [reply docs/pages/integrating/vc.rst:124]: Also numbers and booleans don't support it. They are always exact match

**Review by woutslakhorst — CHANGES_REQUESTED** [review]

File: docs/pages/integrating/vc.rst (hunk 1 above)
Comment (reinkrul) [reply docs/pages/integrating/vc.rst:124]: I'll make it explicit.

**Comment by woutslakhorst** [comment]
Also: breaking changes in release notes

**Review by woutslakhorst — APPROVED** [review]

**Pull request merged**